CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contribution Guidelines
+
+## Issues and Feature Requests
+
+If you got a question, bug report, or feature request, submit an issue to [our repository](https://github.com/DevExpress/devextreme-reactive) but please check the [list of active issues](https://github.com/DevExpress/devextreme-reactive/issues) first to avoid duplicates. 
+
+If you own an active DevExtreme license, feel free to contact us in the [Support Center](https://www.devexpress.com/ask) to receive an answer in shorter time.
```

Comment:
```suggestion
You can use the [Support Center](https://www.devexpress.com/ask) to contact us if you have an active DevExtreme license.
```

Reply:
The fact that we answer quicker to tickets in the Support Center is important

Reply:
If you have an active DevExtreme license, contact us in the [Support Center](https://www.devexpress.com/ask) for an expedited response.